webpack/assets/javascripts/react_app/components/Layout/__tests__/integration.test.js

```diff
@@ -20,26 +20,36 @@ describe('Layout integration test', () => {
     await IntegrationTestHelper.flushAllPromises();
     component.update();
 
-    const yamlLocation = component.find('.location_menuitem');
-    const org2Organization = component.find('.organization_menuitem');
-    const hostsMenuItem = component.find('.secondary-nav-item-pf > a');
     integrationTestHelper.takeStoreSnapshot('initial state');
-
+    const hostsMenuItem = component.find('.secondary-nav-item-pf > a');
+    const locationToggle = component.find(
+      '#location-dropdown .pf-c-context-selector__toggle'
```

Comment:
Is there a way to avoid using the PF class-names and assign our own class-names instead?
So if PF will change there class-names as they did, less chance for breaking changes.

Reply:
line 24: We also use it in the css, can we move this issue to the navigation refactor pr? (https://github.com/theforeman/foreman/pull/7809)
line 26: We can't add a class name to the context selector toggle